package.json

```diff
@@ -51,11 +54,12 @@
     "@mdx-js/react": "1.6.22",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
-    "@storybook/addon-actions": "^6.5.0-beta.0",
+    "@storybook/addon-actions": "^7.0.7",
+    "@storybook/addon-mdx-gfm": "^7.0.7",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/builder-webpack5": "^6.5.0-beta.0",
-    "@storybook/manager-webpack5": "^6.5.0-beta.0",
-    "@storybook/react": "^6.5.3",
+    "@storybook/cli": "^7.0.7",
+    "@storybook/react": "^7.0.7",
```

Comment:
Is this still used somewhere?

Reply:
Clean up a little bit https://github.com/warp-ds/react/pull/27/commits/cc7367baf84f82a7f4bfe679cb034ada2611c3e6 🙌  What do you think?